privval/request_handler.go

```diff
@@ -48,6 +55,11 @@ func hasSignedCommit(valaddr tm_types.Address, commitsigs *[]tm_types.CommitSig)
 	return false
 }
 
+// isRankUpToDate checks whether the validator's rank is still up to date or obsolete.
+func isRankUpToDate(reqHeight int64, LastHeight int64, threshold int) bool {
```

Comment:
why is LastHeight starting with a capital letter?

Reply:
I guess I was bored of lower case params 😆  Fixed.